test/test_helper.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
 require "debug"
-
-require "active_support/testing/strict_warnings"
+begin
+  require "active_support/testing/strict_warnings"
```

Comment:
I don't quite remember what this used to do TBH. I guess it was removed in Rails 8. Maybe ok to just remove the require altogether.